InnerEye/ML/reports/classification_report.ipynb

```diff
@@ -205,8 +221,11 @@
    "outputs": [],
    "source": [
     "if val_metrics_csv.is_file() and test_metrics_csv.is_file():\n",
-    "    print_k_best_and_worst_performing(val_metrics_csv=val_metrics_csv, test_metrics_csv=test_metrics_csv, \n",
-    "                                      k=number_best_and_worst_performing)"
+    "    for hue in config.class_names:\n",
```

Comment:
This means that even for binary classification we get a header "Class=Default", which looks out of place.

Reply:
I think this is not a big problem as use can always rename the class_names